src/core/lib/channel/channelz_registry.cc

```diff
@@ -56,26 +56,28 @@ ChannelzRegistry::~ChannelzRegistry() { gpr_mu_destroy(&mu_); }
 intptr_t ChannelzRegistry::InternalRegisterEntry(const RegistryEntry& entry) {
   MutexLock lock(&mu_);
   entities_.push_back(entry);
-  intptr_t uuid = entities_.size();
+  intptr_t uuid = static_cast<intptr_t>(entities_.size());
   return uuid;
 }
 
 void ChannelzRegistry::InternalUnregisterEntry(intptr_t uuid, EntityType type) {
   GPR_ASSERT(uuid >= 1);
   MutexLock lock(&mu_);
-  GPR_ASSERT(static_cast<size_t>(uuid) <= entities_.size());
-  GPR_ASSERT(entities_[uuid - 1].type == type);
-  entities_[uuid - 1].object = nullptr;
-  entities_[uuid - 1].type = EntityType::kUnset;
+  size_t uuindex = static_cast<size_t>(uuid);
```

Comment:
nit: const

Reply:
also, you may as well store `uuid - 1` instead of `uuid`, given that all access, except for one assert, is over `uuindex -1`.